app/src/main/java/com/myduka/app/app/Config.java

```diff
@@ -26,8 +26,8 @@
 
     //Use credentials from your Lipa na MPESA Online(MPesa Express) App from the developer portal
 
-    public static final String CONSUMER_KEY = "YOUR_CONSUMER_KEY";
-    public static final String CONSUMER_SECRET = "YOUR_CONSUMER_SECRET"; 
+    public static final String CONSUMER_KEY = "YOUR_KEY";
```

Comment:
I'd prefer if these remained as YOUR_CONSUMER_KEY and YOUR_CONSUMER_SECRET for anyone new to Oauth